src/rdiff_backup/selection.py

```diff
@@ -235,14 +247,20 @@ def diryield(rpath):
 
             Returns (rpath, num) where num == 0 means rpath should be
             generated normally, num == 1 means the rpath is a directory
-            and should be included iff something inside is included.
+            and should be included if something inside is included.
```

Comment:
```suggestion
            and should be included iff something inside is included.
```
https://en.wikipedia.org/wiki/If_and_only_if